### DISCUSSION TITLE
kitty sway native wayland

### DISCUSSION THREAD

**Discussion by jenkeeri** [kovidgoyal/kitty#3324] [Q&A]
Hi there!
I`ve recently switched to sway from i3 and I am playing around with my configuration. If my understanding is correct, then kitty should support wayland, but even with setting KITTY_ENABLE_WAYLAND=1, xeyes still show that kitty is run in x11.

I`m on Arch linux using kitty 0.19.3

**Comment by kovidgoyal** [comment]
kitty runs natively under sway automatically unless you set linux_display_server to x11 in kitty.conf

**Reply by jenkeeri** [reply]
Thank you for your answer! I have not set this in my config, but kitty still seems not to run natively under wayland. Maybe you can suggest a different method of testing other than xeyes?

Here is also my full config:

`font_family             SF Mono Powerline Medium
bold_font               SF Mono Powerline Bold
italic_font             SF Mono Powerline Medium Italic
bold_italic_font        SF Mono Powerline Bold Italic
font_size               9

window_padding_width    0 0 2 0
window_margin_width     0

foreground            #f8f8f2
background            #282a36
selection_foreground  #ffffff
selection_background  #44475a

url_color #8be9fd

color0  #21222c
color8  #6272a4

color1  #ff5555
color9  #ff6e6e

color2  #50fa7b
color10 #69ff94

color3  #f1fa8c
color11 #ffffa5

color4  #bd93f9
color12 #d6acff

color5  #ff79c6
color13 #ff92df

color6  #8be9fd
color14 #a4ffff

color7  #f8f8f2
color15 #ffffff

cursor            #f8f8f2
cursor_text_color background

active_tab_foreground   #282a36
active_tab_background   #f8f8f2
inactive_tab_foreground #282a36
inactive_tab_background #6272a4

mark1_foreground #282a36
mark1_background #ff5555`

**Comment by kovidgoyal** [comment]
then set it to wayland to force kitty to use wayland.
